liveblog.php

```diff
@@ -410,9 +416,25 @@ public static function get_liveblog_state( $post_id = null ) {
 		if ( 1 == $state ) {
 			$state = 'enable';
 		}
+		if ( 'enable' == $state && self::$auto_archive_days ) {
+			$time_diff = 60 * 60 * 24 * $auto_archive_days;
+			if ( ( get_post_time( 'U', true, $post_id ) + $time_diff ) < current_time( 'U' ) ) {
```

Comment:
Is this going to match the time the post was published? If so, it's probably better to match on the time of the last Liveblog entry, otherwise a Liveblog that runs for more than 7 days will get unexpectedly archived even though it's active.